Source/assets/serialisers/rbxl/convert_csg.py

```
@@ -2,29 +2,22 @@
 from .. import csg
 
 
-def replace(parser: _logic.rbxl_parser, info: _logic.chunk_info) -> bytes | None:
-    if info.chunk_name == b'SSTR':
-        # TODO: fix CSGv3 SharedString objects.
+def replace(parser: _logic.rbxl_parser, chunk_data: _logic.chunk_data_type) -> _logic.chunk_data_type | None:
+    if isinstance(chunk_data, _logic.chunk_data_type_prop):
+        sstr_data = chunk_data
+        prop_values = _logic.split_prop_strings(sstr_data, len_offset=-4)
```

Ruff (0.12.2)

Local variable `prop_values` is assigned to but never used

Remove assignment to unused variable `prop_values`

(F841)